e2e/wdio/headless/test.e2e.js

```diff
@@ -8,15 +8,27 @@ describe('main suite 1', () => {
     it('should allow to check for PWA', async () => {
         await browser.url('https://webdriver.io')
         await browser.pause(100)
-        expect((await browser.checkPWA()).passed).toBe(true)
+        expect((await browser.checkPWA([
+            'isInstallable',
+            'splashScreen',
+            'themedOmnibox',
+            'contentWith',
+            'viewport',
+            'appleTouchIcon',
+            'maskableIcon'
+        ])).passed).toBe(true)
     })
 
     it('should also detect non PWAs', async () => {
         await browser.url('https://json.org')
         expect((await browser.checkPWA()).passed).toBe(false)
     })
 
-    it('should allow to do performance tests', async () => {
+    /**
+     * fails due to "Unable to identify the main resource"
+     * https://github.com/webdriverio/webdriverio/issues/8541
+     */
+    it.skip('should allow to do performance tests', async () => {
```

Comment:
Is this failing for `v7` as well?

Reply:
Yes, with the same error. So I found that this test skipped in v8 and skip it as well for v7.

Reply:
👍